ci-docker/linux-arm64v8/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM multiarch/ubuntu-core:arm64-bionic
+
+ENV SBT_LAUNCHER_VERSION 1.2.0
+
+RUN apt-get update && apt-get install -y wget
+
+RUN apt-get install -y openjdk-8-jdk-headless
+
+RUN apt-get install -y curl
+
+# Install sbt
+RUN \
+  curl -L -o sbt-$SBT_LAUNCHER_VERSION.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_LAUNCHER_VERSION.deb && \
+  dpkg -i sbt-$SBT_LAUNCHER_VERSION.deb && \
+  rm sbt-$SBT_LAUNCHER_VERSION.deb && \
+  apt-get update && \
+  apt-get install -y sbt
+
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+
+RUN apt-add-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-5.0 main" && apt-get update
+
+RUN apt-get install -y clang-5.0
```

Comment:
For scala-native-bindgen we install `llvm-5.0-dev`. From what I remember package will install `llvm-config` which I know is used by Scala Native. The LLVM tools are located in `/usr/lib/llvm-5.0/bin` so would have to be added to `PATH`.